doc/user_guide/getting_started.rst

```diff
@@ -4,21 +4,15 @@ Getting started
 
 .. _importing_hyperspy-label:
 
-Starting HyperSpy
------------------
+Starting Python in Windows
+----------------------------
+If you used the bundle installation you should be able to use the context menus to get started. Right-click on the folder containing the data you wish to analyse and select "Jupyter notebook here" or "Jupyter qtconsole here". We recommend the former, since notebooks have many advantages over convetional consoles, as will be illustrated in later sections. The examples in some later sections assume Notebook operation. A new tab should appear in your default browser listing the files in the selected folder. To start a python notebook choose "Python 3" in the "New" drop-down menu at the top right of the page. Another new tab will open which is your Notebook.
```

Comment:
> assume Notebook operation

I'm not sure if that's true. I imagine that you want to get rid of the ">>>" to make it less confusing. But those symbols have the nice property that they can be removed from the code examples while the IPython's `In [1]:` can't. Also you could copy and paste the code with the prompts inside the notebook and it'll work. I would rather explain what `>>>` means.